### DISCUSSION TITLE
Delete conversations in Bulk

### DISCUSSION THREAD

**Discussion by j-taptok** [open-webui/open-webui#1346] [Q&A]
The UI is starting to glitch a bit and I suspect the reason to be too many conversations. Is there a way to clean them in bulk?

**Comment by dannyl1u** [comment]
Settings -> Chats -> Delete Chats

**Reply by j-taptok** [reply]
I meant some of the messages about 80%, not all of them.

**Comment by Red007Master** [comment]
As a hack, one can export chats as JSON > {do-your-cleaning} > delete all chats > re-import cleaned JSON.

**Reply by Red007Master** [reply]
Damn, I came here to see my own 'solution', well, if it works.

Tho, proper, 'inside' implementation would be welcome.
